oshdb/src/main/java/org/heigit/ohsome/oshdb/grid/GridOSHNodes.java

```diff
@@ -61,8 +69,8 @@ public OSHNode next() {
         int offset = index[pos];
         int length = (pos < index.length - 1 ? index[pos + 1] : data.length) - offset;
         pos++;
-        return OSHNodeImpl.instance(data, offset, length, baseId, baseTimestamp, baseLongitude,
-            baseLatitude);
+        return OSHNodeImpl.instance(data, offset, length, baseId, baseTimestamp,
+            (int) baseLongitude, (int) baseLatitude);
```

Comment:
True! But at the moment, our test oshdb would then not be compatible any more, because there are already serialized with long and not int.
I will fix this as soon as we recreate our test oshdbs.
Do you want me to do it already in this PR?

Reply:
good point about the test data. it's ok to leave it as it for now. :crossed_fingers: that we don't forget to adapt this then.